### ISSUE TITLE
problems about train code

### ISSUE THREAD

**Issue by zhanghaotian2333** [ZhuangPeiyu/ReLoc#4]
When running the training code with the CASIAv2 dataset, after completing 100 epochs of training, the results on the validation set are generally ineffective. Could this issue be attributed to problems in the publicly available training code, or could it be due to modifications I made to the code?

**Comment by ZhuangPeiyu** [comment]
The newest codes and checkpoints are released.

**Comment by zhanghaotian2333** [comment]
我是张昊天 我已收到您的邮件。有事联系15622707891
